apps/docs/components/column.mdx

```diff
@@ -56,4 +58,4 @@ const Email = () => {
 };
 ```
 
-<Snippet file="support.mdx" />
+<Supporŧ/>
```

Comment:
what's this weird `ŧ` character? 😅

Reply:
LOL, I always end up pressing my right alt + t accidentally and it leaves some `ŧ` characters 😂

Reply:
It's always those characters and the `:w` files LOL